pkg/apis/nfd/nodefeaturerule/expression_test.go

```diff
@@ -265,6 +265,28 @@ func TestEvaluateMatchExpressionValues(t *testing.T) {
 
 		{name: "44", op: nfdv1alpha1.MatchIsFalse, key: "foo", input: I{"foo": "true"}, result: assert.False, err: assert.Nil},
 		{name: "45", op: nfdv1alpha1.MatchIsFalse, key: "foo", input: I{"foo": "false"}, result: assert.True, err: assert.Nil},
+
+		{name: "46", op: nfdv1alpha1.MatchVersionRange, values: V{"", ""}, key: "foo", input: I{"foo": "3.5.0-flavor"}, result: assert.True, err: assert.Nil},
```

Comment:
Should we test and what happens if both min and max are empty strings (`""`) and the input is not valid semver?